### DISCUSSION TITLE
Custom css theme (Bryans Gruvbox.css) oddities / glitches?

### DISCUSSION THREAD

**Discussion by ryanbbrown** [BryanJenksCommunity/FAQ#125] [Q&A] [answered]
Hello! I'm using your custom CSS theme (`Bryans Gruvbox.css`) and like it a lot, but there are a few oddities or glitches that I wanted to see if are possible to fix. I'm on Windows 10. I briefly looked at the code in `Bryans Gruvbox.css` and see that you have very detailed documentation/links for where stuff comes from (thank you for that!) but I'm quite inexperienced with CSS and so thought I'd post them on here in case you (or someone else) more experienced wanted to make these fixes anyways (before I go attempt to dive into it myself). 

**Problem 1:** certain emojis display differently/weirdly, possibly due to a font setting or something with windows 10? All emojis display properly on the normal Gruvbox (`Obsidian gruvbox.css`)
![image](https://user-images.githubusercontent.com/56168848/139952734-01df953e-d7c0-4924-9870-6dec37f5cc4d.png)

**Problem 2:** non top-level bullet points have inconsistent indentation if they have more than one line of text. This isn't a problem with `Obsidian gruvbox.css` (since it doesn't have any bullet point modifications), as shown in the screenshots below:
![image](https://user-images.githubusercontent.com/56168848/139953111-5cbebe1d-41fe-4db1-ada7-41be1edfcaf0.png)
![image](https://user-images.githubusercontent.com/56168848/139953157-c02ff22e-0cd2-4b91-8788-09311322eaeb.png)
![image](https://user-images.githubusercontent.com/56168848/139953623-7af3ae9d-00e3-4369-ac96-bfac8ebc3e2f.png)

**Comment by tallguyjenks** [comment, marked as answer]
1. its the difference between the operating systems/fonts. on my windows machine the emoji's display differently but are still the same emoji if I look at it on my mac
2. it's a know issue with the indentation lines in the CSS. its due to word wrap on the line

**Reply by ryanbbrown** [reply]
**EDIT**: found that 2.4.2 caused justification and commented it out to fix both of the quirks that I was having. The super detailed documentation you have for your custom CSS is HIGHLY appreciated.

I'm now (or maybe I was before and just didn't notice it) having an issue where there's always a slight horizontal scroll on a pane, even if I don't have any other panes open and I have my obsidian app maximized.

![L1ddNrrIto](https://user-images.githubusercontent.com/56168848/140174155-b2da133d-3758-4326-8aca-8383d4cc4ff2.gif)

**Reply by tallguyjenks** [reply]
hold `ctrl` and use the scroll wheel to zoom out, that OR reduce font size OR activate core plugin `readable line length`

**Reply by ryanbbrown** [reply]
turns out it was from the "Embedded Note Titles" plugin--it doesn't cause the scroll bar to appear in default Gruvbox, but must have some weird interaction with your custom CSS. Turning it off gets rid of the issue. Thanks!
